.travis.yml

```diff
@@ -77,6 +81,27 @@ jobs:
       script:
           - make $MAKEJOBS
           - make $MAKEJOBS bench
+    - stage: Memcheck
+      os: linux
+      compiler: gcc
+      env: MEMCHECK=1
+      before_script:
+          - export COVERAGE=OFF
+      script:
+          - make $MAKEJOBS
+          - make $MAKEJOBS ct/_ctcheck
+          - export MAKEFLAGS="valgrind --leak-check=full"
```

Comment:
We can get rid of it by updating to the last version of "ct". I have fixed it there. Let me create patch.

Reply:
PR was created. After the merge we can remove the (kludge) line above.